From 699b1ba55f93719250787cab0d69be82d8b4f245 Mon Sep 17 00:00:00 2001
From: Lasse Heitgres <hello@heitgres.de>
Date: Sun, 25 Jan 2026 13:00:28 +0100
Subject: [PATCH] configs: enable MediaTek MT7925E PCIe Wi-Fi 7

Enable support for the MediaTek MT7925E PCIe Wi-Fi 7 interface.
Contributed by GitHub user @f-io.

See: https://github.com/raspberrypi/linux/issues/7046#issuecomment-3794631559

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/configs/bcm2711_defconfig    | 1 +
 arch/arm64/configs/bcm2711_rt_defconfig | 1 +
 arch/arm64/configs/bcm2712_defconfig    | 1 +
 3 files changed, 3 insertions(+)

--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -633,6 +633,7 @@ CONFIG_MT7601U=m
 CONFIG_MT76x0U=m
 CONFIG_MT76x2U=m
 CONFIG_MT7921U=m
+CONFIG_MT7925E=m
 CONFIG_MT7925U=m
 CONFIG_RT2X00=m
 CONFIG_RT2500USB=m
--- a/arch/arm64/configs/bcm2711_rt_defconfig
+++ b/arch/arm64/configs/bcm2711_rt_defconfig
@@ -633,6 +633,7 @@ CONFIG_MT7601U=m
 CONFIG_MT76x0U=m
 CONFIG_MT76x2U=m
 CONFIG_MT7921U=m
+CONFIG_MT7925E=m
 CONFIG_MT7925U=m
 CONFIG_RT2X00=m
 CONFIG_RT2500USB=m
--- a/arch/arm64/configs/bcm2712_defconfig
+++ b/arch/arm64/configs/bcm2712_defconfig
@@ -636,6 +636,7 @@ CONFIG_MT7601U=m
 CONFIG_MT76x0U=m
 CONFIG_MT76x2U=m
 CONFIG_MT7921U=m
+CONFIG_MT7925E=m
 CONFIG_MT7925U=m
 CONFIG_RT2X00=m
 CONFIG_RT2500USB=m
